tests/foreman/rhci/test_computeresources.py

```diff
@@ -0,0 +1,353 @@
+from robottelo.test import UITestCase
+from robottelo.common import conf
+from robottelo.ui.session import Session
+from robottelo.common.constants import FOREMAN_PROVIDERS
+from robottelo.ui.factory import make_resource
+from ddt import data, ddt
+import time
+
+
+@ddt
+class ComputeResourceTestCase(UITestCase):
+
+    default_org = 'Default Organization'
+    default_loc = 'Default Location'
+
+    vmware_name = conf.properties['vmware.name']
+    vmware_vcenter = conf.properties['vmware.vcenter']
+    vmware_username = conf.properties['vmware.username']
+    vmware_password = conf.properties['vmware.password']
+    vmware_datacenter = conf.properties['vmware.datacenter']
+    vmware_vm_name = conf.properties['vmware.vm_name']
+    vmware_url_name = 'vcenterserver'
+    vmware_dc_locator = 'datacenter_vsphere'
+    vmware_img_name = conf.properties['vmware.img_name']
+    vmware_img_os = conf.properties['vmware.img_os']
+    vmware_img_arch = conf.properties['vmware.img_arch']
+    vmware_img_uname = conf.properties['vmware.img_username']
+    vmware_img_passw = conf.properties['vmware.img_password']
+    vmware_img = conf.properties['vmware.img_image']
+
+    @data(
+        {'name': vmware_name,
+         'provider': FOREMAN_PROVIDERS['vmware'],
+         'url': vmware_vcenter,
+         'username': vmware_username,
+         'password': vmware_password,
+         'datacenter': vmware_datacenter,
+         'url_name': vmware_url_name,
+         'dc_locator': vmware_dc_locator}
+    )
+    def test_create_compute_resource(self, data):
+        """ @Test: Create a compute resource.
+        @Assert: A compute resource is created
+        @Feature: Compute Resource
+        """
+        with Session(self.browser) as session:
+            make_resource(
+                session,
+                name=data['name'],
+                provider_type=data['provider'],
+                parameter_list=[
+                    [data['url_name'], data['url'], 'field'],
+                    ['Username', data['username'], 'field'],
+                    ['Password', data['password'], 'field'],
+                    [data['dc_locator'], data['datacenter'], 'special select']
+                ],
+                orgs=[self.default_org],
+                org_select=False,
+                locations=[self.default_loc],
+                loc_select=True
+            )
+            # nasty hack
+            time.sleep(5)
```

Comment:
yeah, nasty :(
Please, explain why we need it.
`search = self.compute_resource.search(data['name'])` can wait even for 100 hours if needed